repo-transfer.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Transfer a repository to the W3C GitHub organization</title>
+    <link rel="stylesheet" href="css/wgio.css">
+    <link rel="icon" type="image/x-icon" href="//labs.w3.org/favicon.ico">
+  </head>
+  <body>
+    <header>
+      <h1>Transfer a repository to the W3C GitHub organization</h1>
+    </header>
+<nav>
+  <a href="/">Home</a>
+  •
+  <a href="https://github.com/w3c/">Repositories</a>
+  •
+  <a href="https://help.github.com/">GitHub Help</a>
+</nav>
+
+<main>
+  <p>This document describes how to transfer a repository to the W3C GitHub organization.</p>
```

Comment:
```suggestion
  <p>This document describes how to transfer a repository to the <a href="https://github.com/w3c/">W3C GitHub organization</a>.</p>
```

Adding the link as this is the first time it is mentioned (outside of headers).

Reply:
addressed in https://github.com/w3c/w3c.github.io/pull/96/commits/9adbbf4151d916b4c398cfaf82d69d8c37de4bf4